### ISSUE TITLE
Does the minimal code of Cdpam loss provided is a trained model or not trained?

### ISSUE THREAD

**Issue by ericwudayi** [pranaymanocha/PerceptualAudio#19]
Hi, Does the minimal code of Cdpam loss provided is a trained model or not trained?:

I have tried this:

For denoising: if modelA is a denoising model then

import cdpam
loss_fn = cdpam.CDPAM()
wav_syn = modelA(wav_noise)
loss = loss_fn.forward(wav_true, wav_syn)
loss.backward() --> update modelA's parameter

I expect that modelA would learn something. Am I right?

**Comment by pranaymanocha** [comment]
Hi, Yes this is correct. However, we think that this loss should be applied in conjunction with other losses like L1 / multiresolution STFT etc. I am planning to post the paper on arxiv soon, but the idea for now is for this loss to work well along with other conventional losses to remove perceptible artifacts not captured by other losses. This is especially true at the beginning of training (like the denoiser that you mentioned). Hope this clarifies things!

**Comment by ericwudayi** [comment]
Thank you very much!
